pydantic/main.py

```diff
@@ -414,14 +414,17 @@ def from_orm(cls: Type['Model'], obj: Any) -> 'Model':
         return m
 
     @classmethod
-    def construct(cls: Type['Model'], values: 'DictAny', fields_set: 'SetStr') -> 'Model':
+    def construct(cls: Type['Model'], _fields_set: Optional['SetStr'] = None, **values: Any) -> 'Model':
```

Comment:
@samuelcolvin Would it make sense to rename `cls` to `__pydantic_cls__` (similar to in `BaseModel.__init__`), or at least something that starts with a `_` to prevent keyword argument conflicts?